includes/checkout/template.php

```diff
@@ -225,62 +225,66 @@ function edd_user_info_fields() {
 function edd_get_cc_form() {
 	ob_start(); ?>
 
-	<?php do_action( 'edd_before_cc_fields' ); ?>
+		<?php do_action( 'edd_before_cc_fields' ); ?>
 
 	<fieldset id="edd_cc_fields" class="edd-do-validate">
-		<legend><?php _e( 'Credit Card Info', 'easy-digital-downloads' ); ?></legend>
+		<legend><?php esc_html_e( 'Credit Card Info', 'easy-digital-downloads' ); ?></legend>
+
 		<?php if( is_ssl() ) : ?>
 			<div id="edd_secure_site_wrapper">
 				<span class="padlock">
```

Comment:
@cklosowski I'm not sure if I'm supposed to have .padlock, which gets icon on :before and the inline SVG or not. Please let me know if this is correct. I'm not testing with SSL.